### DISCUSSION TITLE
Again, problems with updating paperless, this time to v2.11.0

### DISCUSSION THREAD

**Discussion by Tropaion** [tteck/Proxmox#3358] [General] [answered]
Lately I often have problems with updating paperless using the script, this time it says this:
![image](https://github.com/tteck/Proxmox/assets/6797663/6b330368-215f-480e-a6d9-56ceba03f0e4)
Any ideas what I can do?

**Comment by tteck** [comment, marked as answer]
https://github.com/tteck/Proxmox/discussions/3300

**Reply by Tropaion** [reply]
That worked, thanks 😄
